### PR TITLE
Added bio + picture

### PR DESCRIPTION
**Pull request by esavary**
- Added a bio + additional information (Not sure about the title) in team_members.yml
- Added a cropped picture in images\teampic

### CODE REVIEW

File: _data/team_members.yml [hunk 1]
```diff
@@ -55,3 +55,21 @@
 
   links:
     - '[ORCID](https://orcid.org/0000-0002-1668-9629)'
+
+- name: Elodie Savary
+  photo: photo_esavary300x300.png
+  email: elodie.savary@outlook.com
+  title: Developer
```
Comment (oesteban) [inline _data/team_members.yml:62]:
```suggestion
  title: Scientific software developer 
```

File: _data/team_members.yml [hunk 2]
```diff
@@ -55,3 +55,21 @@
...
+  email: elodie.savary@outlook.com
+  title: Developer 
+  education:
+    - PhD. (2022) EPFL (Lausanne, Switzerland)
+    - M.Sc. (2018) EPFL (Lausanne, Switzerland)
+    - B.Sc. (2016) EPFL (Lausanne, Switzerland)
+  bio: |
+    My background is in Astrophysics. During my PhD, I developed deep-learning tools to classify, segment and generate images.
+    Besides the machine learning components, my PhD research also included a significant amount of image processing.
+    I will contribute to the implementation of preprocessing tools to improve computational reproducibility and reduce methodological variability in MRI research.
```
Comment (oesteban) [inline _data/team_members.yml:70]:
```suggestion
    In The AxonLab, I contribute to implementing preprocessing tools under the umbrella of the [*NiPreps* ecosystem](https://nipreps.org) to improve computational reproducibility and reduce methodological variability in MRI research.
```

**Review by oesteban — COMMENTED** [review]
Very minor suggestions here and there, but overall, LGTM (looks good to me)

**Comment by esavary** [comment]
Thanks a lot for the suggestions! I included them in my last commit.

**Pull request merged**